src/foam/comics/v2/DataSummaryCard.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.comics.v2',
+  name: 'DataSummaryCard',
+  extends: 'foam.u2.View',
+
+  documentation: `
+    A configurable view to to render a card with 
+    configurable contents and rich choice view dropdowns
+  `,
+
+  css:`
+  `,
+
+  requires: [
+    'foam.u2.layout.Cols',
+    'foam.u2.layout.Rows',
+    'foam.u2.ControllerMode',
+    'foam.u2.borders.CardBorder'
+  ],
+  exports: [
+    'controllerMode'
+  ],
+  properties: [
+    {
+      class: 'FObjectProperty',
+      name: 'data'
+    },
+    {
+      name: 'controllerMode',
+      factory: function() {
+        return this.ControllerMode.VIEW;
+      }
+    },
+    {
+      class: 'FObjectArray',
+      of: 'Reference',
+      name: 'choices',
+      factory: function() {
```

Comment:
Isn't this already the default factory for FObjectArray?

Reply:
Removed.